### PR TITLE
Add Confirmation Dialog Pattern 

### PR DESCRIPTION
**Pull request by deernano**
## Description
* Add a standard pattern for confirmation dialogs.
* #632 

## Changes

Write a list of changes this design pattern introduces

* Change 01 Add text and images in Overview Page
* Change 02 Add text and images in Design Page
* Change 03 Add images in Image folder
* Change 04 Find a small title mistake in Wizard Design page. Updated

## Contribution Submission Checklist

- [X] A GitHub issue was created and reviewed with the PatternFly community.
- [X] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [X] Any new files have been added to the folder for that pattern.
- [X] The folder, for brand new patterns, has been titled with the name of the pattern.
- [X] All images are in an "img" folder and embedded images have the correct filepath.
- [X] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by deernano** [comment]
@mcarrano 'Also, looks like you copied the site.md from Wizard but forgot to update the title and links here to point to this new pattern.'----This has been updated.
In this PR,  there are only two section  Usage and the Message Dialog Variants.  The Elements of a Message Dialog in the sub section under Usage. The Message Dialog, Message Dialog with Status Icon,Status Icons for Message Dialogs.  Do you mean every sub section should go into Design Page?

**Comment by mcarrano** [comment]
@deernano You may have misunderstood my comment about the site.md. The contents of the file itself needs to be edited. If you open this, you will see that the links are pointing to the Wizard. You need to edit this to point to the files you are adding. I think that @dabeng is familiar with this if you need help.

As for the content itself, this looks better. I'm thinking that the table of valid icons belongs on the Design rather than the Overview page. Thoughts?

**Comment by deernano** [comment]
@mcarrano 
Update the site.md to the correct link.
Move the status icon to Design page.
Please take a look again, Thanks~

File: pattern-library/communication/message-dialog/design/design.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+![Error Dialog 1](img/msg-dialog-error1@2x.png)
+![Error Dialog 2](img/msg-dialog-error2@2x.png)
+
+Never blame the user. Use passive voice where appropriate to avoid assigning blame.
+- ⛔ You put in the wrong password.
+- ✅ That password wasn’t recognized OR
+- ✅ We didn’t recognize that password.
+
+### Passive Dialog
+Use passive dialogs to communicate critical information like the status of an application or result of an action. Passive modals should clearly communicate the issue and any actions required of the user as a result of the issue.Information in a passive dialog should be important, immediately relevant, and either lead to or require action on the users part.
```
[the remark this answers is not in the document]
Comment (deernano) [reply pattern-library/communication/message-dialog/design/design.md]: Updated

[the remark this answers is not in the document]
Comment (deernano) [reply pattern-library/communication/message-dialog/design/design.md]: Updated

File: pattern-library/communication/message-dialog/design/design.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
+# Message Dialogs
+
+## Elements of a Message Dialog
+![Message Dialog Elements](img/msg-dialog-elements@2x.png)
+
+1. **Title bar (Required):** Titles are be informative and effectively group the content in the dialog. Do not use general or ambiguous statements like alert, warning or Are you sure?
```
[the remark this answers is not in the document]
Comment (deernano) [reply pattern-library/communication/message-dialog/design/design.md]: Updated

**Pull request merged**